Sources/.swiftlint.yml

```diff
@@ -46,17 +47,16 @@ identifier_name:
   allowed_symbols: ["_"]
   validates_start_with_lowercase: false
 
-line_length: 
-  warning: 134
+line_length:
+  warning: 132
   ignores_comments: true
+  ignores_urls: true
 
 cyclomatic_complexity:
   ignores_case_statements: true
   warning: 25
   error: 30
 
-file_length: 510
+file_length: 500
```

Comment:
I'd suggest removing this. Sometimes it's helpful to have longer files. I'd leave this at the developer's opinion.

Reply:
@buh ☝️ ??

Reply:
I think this value doesn’t work because I excluded the rule.